src/test/resources/org/jetbrains/research/transformations/examplePyCode/test_1.py

```diff
@@ -0,0 +1,2 @@
+a = 3
```

Comment:
Do you use it somewhere? I cannot find any usages, why should we have it in master?